docs/using/syntax.md

```diff
@@ -434,6 +435,26 @@ Is below, but it won't be parsed into the document.
 
 % my comment
 
+(syntax/blockbreaks)=
+
+### Block Breaks
+
+You may add a block break by putting `+++` at the beginning of a line.
+This constuct's intended use case is for mapping to cell based document formats,
+like [jupyter notebooks](https://jupyter.org/),
+to indicate a new text cell. It will not show up in the rendered text,
+but is stored in the internal document structure for use by developers.
+
+For example, this code:
+
+```
++++ some text
```

Comment:
The "some text" doesn't make it into the AST in a structured way, right? And if not I don't think we should add it here just to make sure users don't think it's usable in some way

Reply:
Yeh it does, have a look at the examples in my comment, where "a block break" is stored. I was thinking you might want to use it to serialize and 'store' text cell metadata, for round-tripping (even if this metadata is not actually used in sphinx), e.g.

```
+++ {"key": "value"}
```